tests/fixtures/smoke-testing.ts

```diff
@@ -55,6 +57,15 @@ class PuppetTest extends Puppet {
   public async contactSelfName (name: string)           : Promise<void> { return }
   public async contactSelfSignature (signature: string) : Promise<void> { return }
 
+  /**
+   *
+   * Tag
+   *
+   */
+  public async createTag (tag: string)           : Promise<TagPayload> { return { tag } as any }
+  public async addTag (to: string)               : Promise<void> { return { to } as any }
+  public async deleteTag (from: string)          : Promise<void> { return { from } as any }
```

Comment:
These two should return nothing, since the declaration of the function is `void`.